lib/service.js

```diff
@@ -149,7 +154,7 @@ class Service {
       .exec()
       .then(data => {
         if (!data) {
-          throw new errors.NotFound(`No record found for id '${id}'`);
+          throw new errors.NotFound(`No record found for id '${query[this.id]}'`);
```

Comment:
There’s a possibility, now, that there won’t be any `id ’ at this point.  If you do `service.get(null, {test: true})`, for example.  If that’s the case, I think we should just say “No record found”

Reply:
To be clear, I think the current message is ideal as long as we have an id.